lib/Map/ImageryProvider/ProtomapsImageryProvider.ts

```diff
@@ -248,50 +232,52 @@ export default class ProtomapsImageryProvider
     return [];
   }
 
-  async requestImage(x: number, y: number, level: number) {
+  async requestImage(x: number, y: number, level: number, request: Request) {
     const canvas = document.createElement("canvas");
     canvas.width = this.tileWidth;
     canvas.height = this.tileHeight;
-    return await this.requestImageForCanvas(x, y, level, canvas);
+    return await this.requestImageForCanvas(x, y, level, canvas, request);
   }
 
   async requestImageForCanvas(
     x: number,
     y: number,
     level: number,
-    canvas: HTMLCanvasElement
+    canvas: HTMLCanvasElement,
+    request?: Request | undefined
   ) {
-    try {
-      await this.renderTile({ x, y, z: level }, canvas);
-    } catch (e) {
-      console.log(e);
-    }
+    if (this.softMinimumLevel && level < this.softMinimumLevel)
+      throw TerriaError.from(`Level: ${level} is below softMinimumLevel`);
```

Comment:
Can the value of `this.softMinimumLevel` be seen when this error is triggered, or should that be included in the error message?